pilot/pkg/xds/xdsgen.go

```diff
@@ -155,6 +157,118 @@ func (s *DiscoveryServer) pushXds(con *Connection, w *model.WatchedResource, req
 	return nil
 }
 
+// Push an Delta XDS resource for the given connection. Configuration will be generated
+// based on the passed in generator.
+func (s *DiscoveryServer) pushDeltaXds(con *Connection,
```

Comment:
why move out of Delta code?

Reply:
I think before delta stable we should keep it separate， after it is mature we can merge it with Stow, so reuse some code